### ISSUE TITLE
'darknet.exe export ' doesn't work in Windows version

### ISSUE THREAD

**Issue by B1ackg0d** [AlexeyAB/darknet#8004]
![image](https://user-images.githubusercontent.com/50610732/129440479-2d23a852-77a3-4e59-a825-1d2ed079647f.png)

Is the way I am using it wrong?
I want to use darknet to export weights on windows, and then use tkdnn-winows to generate rt files.
Could you help me?Thx!

**Comment by stephanecharette** [comment]
The [README](https://github.com/AlexeyAB/darknet/blob/master/README.md) has all the commands, and I don't see "export" being one of them.  What made you think that was something you can run, and what did you think it would do?

**Comment by B1ackg0d** [comment]
I’d like to apologize.I'm a novice. Because there is such a function as export(`./darknet export .\cfg\yolov4.cfg .\yolov4.weights layers`) in Linux, so I thought there would be similar functions in Windows. I just want to use darknet to export the weights of yolo4 in Windows.Could you give me some good advice？Or start to modify the direction of the source code.Thanks for your reply.

**Comment by stephanecharette** [comment]
I don't know what to tell you.  As far as I know there is no such function, not in Linux, and not in Windows.  Please read through the readme file I linked above.  I have no idea what you mean by "export the weights of yolo4".

```
~/src/darknet/darknet export
Not an option: export
```

**Comment by B1ackg0d** [comment]
https://github.com/ceccocats/tkDNN/blob/master/docs/exporting_weights.md 
Isn't the darknet mentioned here the same thing?I am also a bit confused

**Comment by stephanecharette** [comment]
No, I've never heard of that project.  You are posting this issue on https://github.com/AlexeyAB/darknet/ but you are asking about https://github.com/ceccocats/tkDNN/ which references another tool hosted at https://git.hipert.unimore.it/fgatti/darknet.git.
Neither ceccocats/tkDNN nor hipert.unimore.it/fgatti have anything to do with this project.

**Comment by engcang** [comment]
Hi. @B1ackg0d @stephanecharette 

@AlexeyAB has mentioned `export` function for using with `tkDNN` before.
See here - https://github.com/AlexeyAB/darknet/issues/6116#issuecomment-655483646
Or, you can directly download the version with `export` here - https://github.com/AlexeyAB/darknet/files/4890564/darknet-master.zip
